spec/models/manageiq/providers/amazon/aws_refresher_spec_counts.rb

```diff
@@ -65,15 +65,11 @@ def table_counts_from_api
     # Custom attributes of all Vms and Images
     custom_attributes_count = (vms_tags + images_tags).size
 
-    vms_mappings = ProviderTagMapping.where(:labeled_resource_type => nil).or(
```

Comment:
I'm not sure where `:labeled_resource_type => nil` came from, as far as I know the "all entities" started with using `"_all_entities_"` not `nil`